package.xml

```diff
@@ -80,4 +80,39 @@
   </dependencies>
   <providesextension>smbclient</providesextension>
   <extsrcrelease/>
+  <changelog>
```

Comment:
Just curious as to why you've added the missing changelog but not one for 0.9.0dev you are adding.